src/@next/Badge/BadgeStyle.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { S08 } from '../utilities/colors/blue';
+import { B89 } from '../utilities/colors/green';
+import { B95 } from '../utilities/colors/neutral';
+import { S42, S87 } from '../utilities/colors/orange';
+import { B100 } from '../utilities/colors/red';
+import { S75 } from '../utilities/colors/yellow';
+import { space8 } from '../utilities/spacing';
+import { BadgeProps, BadgeStatusType } from './Badge';
+
+const badgeBackgroundColor = {
+  success: B89,
+  informational: S08,
+  warning: S42,
+  critical: B100,
+  promotion: S75,
+  new: S87,
+};
+
+const getBadgeBackgroundColor = (status: BadgeStatusType) => {
+  if (!(status in badgeBackgroundColor)) {
+    return B95;
```

Comment:
we should probably warn user if status is not found similar to the one in Button https://github.com/glints-dev/glints-aries/blob/c539920df36a3c65a2cc3eb57d243148d6eef0eb/src/%40next/Button/ButtonStyle.ts#L108

Reply:
i don't think we should warn the issue for this case because it's default behaviour; user should expect this color if they don't pass in any attribute

Reply:
we still would want to warn the user if they passed an incorrect status though. it would defeat the purpose of having status types if they passed a different one and they would wonder why the color is of default. it would make it easier for dev to know. (additional context regarding default type [here](https://github.com/glints-dev/glints-aries/pull/709#discussion_r1067863131))